__tests__/cmds/versions.test.js

```diff
@@ -103,30 +105,36 @@ describe('rdme versions*', () => {
     });
 
     it('should create a specific version', async () => {
-      promptHandler.createVersionPrompt.mockResolvedValue({
-        is_stable: true,
-        is_beta: false,
-        from: '1.0.0',
-      });
-
       const mockRequest = getApiNock()
-        .get('/api/v1/version')
-        .basicAuth({ user: key })
-        .reply(200, [{ version }, { version }])
-        .post('/api/v1/version')
+        .post('/api/v1/version', {
+          version: '1.0.0',
+          codename: '',
+          is_stable: false,
+          is_beta: false,
+          from: '1.0.0',
+          is_hidden: true,
+        })
         .basicAuth({ user: key })
         .reply(201, { version });
 
-      await expect(createVersion.run({ key, version })).resolves.toBe('Version 1.0.0 created successfully.');
+      await expect(createVersion.run({ key, version, fork: '1.0.0', main: true, beta: false })).resolves.toBe(
+        'Version 1.0.0 created successfully.'
+      );
+
       mockRequest.done();
     });
 
+    it.todo('should prompt a version list if no `--fork` argument is supplied');
```

Comment:
Not having something like `promptHandler.createVersionPrompt.mockResolvedValue` to mock Enquirer responses with this new setup is annoying but as it is I'm comfortable with the tradeoff we're making with this work and not having to have tests for Enquirer being able to prompt us for questions.

Reply:
lmaoo yeah I like that we can mock and reuse responses with that `promptHandler`, but we have a lot of different patterns in that file that are pretty confusing so I think we should revisit that at some point